modules/google-pubsub/package.json

```diff
@@ -19,27 +23,33 @@
   },
   "license": "MIT",
   "peerDependencies": {
-    "@nestjs/common": "^8.0.0",
-    "typescript": ">=3.9.9"
+    "@nestjs/common": ">=8.0.0",
+    "typescript": ">=4.0.0"
   },
   "dependencies": {
-    "@google-cloud/pubsub": "1.6.0",
+    "@erento/eslint-plugin-erento-rules": "1.0.0",
+    "@google-cloud/pubsub": "2.18.5",
     "message-crypto": "1.0.0",
-    "object-hash": "2.0.3"
+    "object-hash": "2.2.0"
   },
   "devDependencies": {
     "@nestjs/common": "8.3.0",
-    "@types/jest": "25.1.4",
-    "@types/node": "12.12.30",
-    "husky": "4.2.3",
-    "jest": "25.1.0",
-    "lint-staged": "10.0.9",
-    "prettier": "1.19.1",
+    "@types/jest": "27.4.0",
+    "@types/node": "17.0.18",
```

Comment:
If you really want to use node 16 as a minimum, I would also in the code only use types form the minum version and not higher. The types here should match the minimum version required for the package to run.